src/main/java/emissary/place/CoordinationPlace.java

```diff
@@ -220,49 +215,124 @@ protected List<IBaseDataObject> coordinate(IBaseDataObject d, boolean hd) {
                 }
                 errorOccurred = d.currentForm().equals(Form.ERROR);
             } catch (Exception ex) {
-                logger.warn("agentProcess {} called from Coordinate problem", (hd ? "HeavyDuty" : "Call"), ex);
-                errorOccurred = true;
+                errorOccurred = handlePlaceException(hd, ex);
             } finally {
                 if (Thread.interrupted()) {
                     logger.warn("Place {} was interrupted during execution.", p);
                 }
             }
 
-            if (errorOccurred) {
-                logger.info("Error terminating coordination step at {}", p);
+            if (shouldNotContinueOnError(p, errorOccurred))
                 break;
-            }
 
             // Track any new attachments
             if (CollectionUtils.isNotEmpty(sprouts)) {
                 sproutCollection.addAll(sprouts);
             }
         }
 
-        if (!errorOccurred) {
-            // Process the ouptut form according to configuration
-            if (outputForm != null) {
-                if (pushForm) {
-                    d.pushCurrentForm(outputForm);
-                } else {
-                    d.setCurrentForm(outputForm);
-                }
-            }
+        processForm(d, errorOccurred);
 
-            // Clean up my proxies
-            nukeMyProxies(d);
+        sproutHook(d, hd, sproutCollection);
 
-        }
+        // Allow derived classes a shot to clean up the parent
+        cleanUpHook(d);
 
+        return sproutCollection;
+    }
+
+    private void sproutHook(IBaseDataObject d, boolean hd, List<IBaseDataObject> sproutCollection) {
         // Allow derived classes a shot at the sprouts
         if (hd) {
             sproutHook(sproutCollection, d);
         }
+    }
 
-        // Allow derived classes a shot to clean up the parent
-        cleanUpHook(d);
+    /**
+     * Allow derived classes a shot to handle a place exception
+     *
+     * @param hd true if doing heavy-duty processing
+     * @param ex exception thrown by the place
+     *
+     * @return if an error occurred
+     */
+    protected boolean handlePlaceException(boolean hd, Exception ex) {
+        logger.warn("agentProcess {} called from Coordinate problem", (hd ? "HeavyDuty" : "Call"), ex);
+        return true;
+    }
 
-        return sproutCollection;
+    /**
+     * Return true when an error has occurred and the places is configured to not continue
+     *
+     * @param p place that is currently processing the ibdo
+     * @param errorOccurred true if an error occurred
+     *
+     * @return true if processing should not continue
+     */
+    private boolean shouldNotContinueOnError(IServiceProviderPlace p, boolean errorOccurred) {
+        if (allowShouldNotContinueOnError() && errorOccurred) {
```

Comment:
I'm not sure this has to exist as part of the override. I could see wanting to override the private method in this case to do something other than the default scenario. 

`protected boolean shouldContinue(p, errorOccurred)`

`protected boolean continueOnError()`